Sources/Error Handling/StoreKitErrorHelper.swift

```diff
@@ -27,9 +27,9 @@ enum StoreKitErrorUtils {
             return storeKitError.trackingDescription
         } else if let storeKitError = underlyingError as? StoreKit.Product.PurchaseError {
             return storeKitError.trackingDescription
+        } else {
+            return Self.extractStoreKitErrorDescription(from: underlyingError)
```

Comment:
In case the current underlying error is not a store kit error, we try to get the underlying error for the current error, if it exists. I don't think this should result in an infinite loop, as long as we don't have a circular reference in the underlying errors.

Reply:
> I don't think this should result in an infinite loop

Yeah, I agree. But still, this scares me a bit. However, I honestly can't think of a way to check this without overcomplicating things a lot 🤔